### ISSUE TITLE
[Bug] White spaces in folders names

### ISSUE THREAD

**Issue by Kristinita** [beautify-web/js-beautify#1162]
### 1. Summary

I want to use js-beautify for recursively prettify all `html` files of my project. But files in folders, contains whitespaces, don't prettify for me. Possibly, it a bug.

### 2. Settings

I run html-beautify commands via [**glob-run module**](https://github.com/fahad19/glob-run) like [**Stack Overflow answer**](http://stackoverflow.com/a/33152801/5951529).

    glob-run html-beautify -r **/*.html

For example, in directory of my project I have files ` Smert-svobode/Новые-недовольные-русские.html` and `Sublime Text/Sasha-Bozhestvenna.html`.

### 3. Expected behavior

For `Smert-svobode/Новые-недовольные-русские.html`:

```javascript
>> html-beautify -r Smert-svobode/Новые-недовольные-русские.html
beautified Smert-svobode\Новые-недовольные-русские.html - unchanged
```

File successful prettify for me.

### 4. Actual behavior

For `Sublime Text/Sasha-Bozhestvenna.html`:

```javascript
>> html-beautify -r Sublime Text/Sasha-Bozhestvenna.html
Unable to open path "D:\Kristinita.bitbucket.org\output\Sublime"
Run `html-beautify.js -h` for help.
exec error: Error: Command failed: html-beautify -r Sublime Text/Sasha-Bozhestvenna.html
Unable to open path "D:\Kristinita.bitbucket.org\output\Sublime"
Run `html-beautify.js -h` for help.
```

File don't prettify for me.

### 5. Steps to reproduce

In head directory of my project I run command `glob-run html-beautify -r **/*.html`. I get expected behavior, if folder names not contains whitespaces; I get actual behavior, if folder names contains whitespaces.

### 6. Environment

**Operating system and version:**
Windows 10 Enterprise LTSB 64-bit EN
**Node.js:**
v7.7.3
**html-beautify:**
1.6.11
**glob-run:**
0.1.6

Thanks.

**Comment by bitwiseman** [comment]
This seems like a bug in `glob-run` it is likely calling the module without quotes around the parameters.

If you manually call `html-beautify -r Sublime Text/Sasha-Bozhestvenna.html` you will get the same result.  

If you call `html-beautify -r 'Sublime Text/Sasha-Bozhestvenna.html'`, it will work.

**Comment by Kristinita** [comment]
@bitwiseman , but how Windows users can run `js-beautify` for multiple files recursively?

`glob-run` no longer maintained, that fix this bug. [**This**](http://stackoverflow.com/a/23258950/5951529) and [**this**](http://stackoverflow.com/a/23254049/5951529) answers for Linux users.

Thanks.

**Comment by bitwiseman** [comment]
@Kristinita  You can totally still run this on windows.  
Simplest workaround is to pick a folder tree and files that don't have spaces in them.
Or you could fix #787.

**Comment by Kristinita** [comment]
@bitwiseman , all right, I replace all whitespaces in files and folders names in my repository.

But how I can ignore some folders? For example, I don't want to prettify content of my [**bower-components folder**](https://github.com/Kristinita/Kristinita.github.io/tree/master/theme/bower-components).  I don't find this information in [**js-beautify options**](https://github.com/beautify-web/js-beautify#options).

Thanks.
